app/models/manageiq/providers/openstack/inventory_collection_default/cloud_manager.rb

```diff
@@ -0,0 +1,269 @@
+class ManageIQ::Providers::Openstack::InventoryCollectionDefault::CloudManager < ManagerRefresh::InventoryCollectionDefault::CloudManager
+  class << self
+    def availability_zones(extra_attributes = {})
+      attributes = {
+        :inventory_object_attributes => [
+          :type,
+          :ems_ref,
+          :name
+        ]
+      }
+      super(attributes.merge!(extra_attributes))
+    end
+
+    def cloud_resource_quotas(extra_attributes = {})
+      attributes = {
+        :model_class                 => CloudResourceQuota,
+        :association                 => :cloud_resource_quotas,
+        :manager_ref                 => [:ems_ref],
```

Comment:
:manager_ref  => [:ems_ref] is a default value, so you don;t probably need to specify that